.github/workflows/ci.yml

```diff
@@ -36,6 +36,40 @@ jobs:
       - name: Run clippy with all features
         run: cargo clippy --workspace --all-targets --all-features -- -D warnings
 
+  ui_e2e_test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Set up environment and build binaries
+        run: |
+          cd waves
```

Comment:
FYI: There is a `working-directory` option: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsrun

Reply:
You can also set the `working-directory` for the whole job: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_iddefaultsrun